oesukam/lit-blogpost
# #163520287 implement post comment feature

### What does this PR do?
This PR will add comments endpoint to system

### Description of Task to be completed?
Checking if a post exists
Checking if a post is published
Add a comment to a post

### How should this be manually tested?
npm install
connect Database
$ sequelize db:migrate
check on `http://localhost:3000/api/v1/posts/postId/comments` with POST method

### What are the relevant pivotal tracker stories?
#1163520287
